addOns/ascanrules/CHANGELOG.md

```diff
@@ -11,10 +11,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ### Changed
 - SQL Injection scan rule to start using ComparableResponse - part of the work to reduce False Positives.
 - Depends on an updated version of the Common Library add-on.
+- De-prioritized plain HTTP payloads in the External Redirect scan rule.
```

Comment:
Be good to justify this change, for the record.

Reply:
If you need the stats to justify the change.

Reply:
I tweaked the log entry.

I also quickly looked at stats top few hits puts HTTPS at 88% or more of web traffic.